custom_components/tuya_local/devices/caldo_up_t_wifi.yaml

```diff
@@ -28,11 +26,11 @@ primary_entity:
       type: string
       name: preset_mode
       mapping:
-        - dps_val: "0"
-          value: none
-        - dps_val: "1"
+        - dps_val: "level_1"
+          value: "fan only"
```

Comment:
fan_only is not a standard preset name. I think this is better left as "none", which conveys that compared to "comfort" and "boost", it provides no heating.